AmazingMeSample/app/src/main/java/amazingme/activities/app/RegisterActivity.java

```diff
@@ -0,0 +1,60 @@
+package amazingme.activities.app;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import com.amazingme.activities.R;
+
+import amazingme.app.EnumeratedActivity;
+import amazingme.database.FirebaseHelper;
+import amazingme.model.AmazingMeAppCompatActivity;
+
+/**
+ * Created by yudawinata on 8/31/17.
```

Comment:
let's trash our "created by" comments. 

i tried to be good about that in mine, but i might have missed some (let me know if you see them and i'll get rid of them). i don't think we need it (we can see who made it in the commit and i'd rather put something about the CDC or AmazingMe in it's place if we really want it

Reply:
sure.